NTO/Automation/attributes/parameters.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix ogit.Automation:        <http://www.purl.org/ogit/Automation/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Automation:parameters
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "parameters";
+	dcterms:description "Contains a parameters in a json map, where key is a parameter name and value is a value of a parameter.";
```

Comment:
Here I would say we can safely remove the json map reference and just specify key/value semantics in the list values directly

Reply:
one open question here would be how to distinguish between mandatory and optional parameters... could be done either by splitting into "required_parameters" and "optional_parameters" or by defining special values that need to be used for "needs to be specified by user" (simply having empty value could not be distinguished from empty default)